students/11th/daseulsong/js/login.js

```diff
@@ -0,0 +1,20 @@
+'user strict';
+
+const loginBox = document.querySelector('.login__main__idPw');
+const id = document.querySelector('.login__idPw--id');
+const pw = document.querySelector('.login__idPw--pw');
+loginBtn = document.querySelector('.login__btn--blue');
+[id, pw].forEach((item) => {
+  console.log(item);
+  item.addEventListener('keyup', (event) => {
+    console.log(event);
```

Comment:
```suggestion
  item.addEventListener('keyup', () => {
```
1. forEach 문 잘 써주셨습니다! 👏 
2. 테스트 끝난 console.log 삭제해주세요
3. 2번째 인자에 위치하는 함수에서 event 객체가 활용되는 부분이 없는 것 같습니다. 삭제해주세요.

Reply:
테스트용 코드와 event 모두 삭제 완료했습니다.